### PR TITLE
Org management - Organization View improvements

### PR DESCRIPTION
**Pull request by kavinda1995**
* Fix organization description max, min length issue
* Fix Organization update patch request sent twice
* Fix UX issue when deleting an organization with sub organizations
* Add organization ID to organization list view and organization switcher to improve UX when having organizations with same names

### Related Issues
- https://github.com/wso2/product-is/issues/14364
- https://github.com/wso2/product-is/issues/14391
- https://github.com/wso2/product-is/issues/14349
- https://github.com/wso2/product-is/issues/14366

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/organizations/components/edit-organization/organization-overview.tsx [hunk 1]
```diff
@@ -187,6 +214,24 @@ export const OrganizationOverview: FunctionComponent<OrganizationOverviewPropsIn
     }, [ organization ]
     );
 
+    const validate = async (values: OrganizationEditFormProps): Promise<Partial<OrganizationEditFormProps>> => {
+        const error: Partial<OrganizationEditFormProps> = {};
+
+        if (values?.name && (values.name.length < ORGANIZATION_NAME_MIN_LENGTH
+            || values?.name.length > ORGANIZATION_NAME_MAX_LENGTH)) {
+            error.name = `Organization name length should be at least ${ ORGANIZATION_NAME_MIN_LENGTH } `
```
Comment (thivi) [inline apps/console/src/features/organizations/components/edit-organization/organization-overview.tsx:222]: Better to localize this

**Review by thivi — APPROVED** [review]

File: apps/console/src/features/organizations/components/organization-list.tsx [hunk 2]
```diff
@@ -1,7 +1,7 @@
 /**
- * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```
Comment (NipuniBhagya) [inline apps/console/src/features/organizations/components/organization-list.tsx:2]: Let's not update the license year of the older files.

File: apps/console/src/features/organizations/components/organization-switch/organization-switch-dropdown.tsx [hunk 3]
```diff
@@ -239,30 +239,39 @@ const OrganizationSwitchDropdown: FunctionComponent<OrganizationSwitchDropdownIn
...
+                                    { organization?.name ?? (
+                                        <Placeholder>
+                                            <Placeholder.Line />
+                                        </Placeholder>
+                                    ) }
+                                </div>
+
+                                <Label size="mini">{ organization.id }</Label>
+                            </div>
+
```
Comment (NipuniBhagya) [inline apps/console/src/features/organizations/components/organization-switch/organization-switch-dropdown.tsx:253]: Let's remove unnecessary new lines.

Comment (kavinda1995) [reply apps/console/src/features/organizations/components/organization-switch/organization-switch-dropdown.tsx:253]: Latest changes removed this section of the component.

File: apps/console/src/features/organizations/components/organization-list.tsx (hunk 2 above)
Comment (kavinda1995) [reply apps/console/src/features/organizations/components/organization-list.tsx:2]: Fixed in https://github.com/wso2/identity-apps/pull/3235

**Pull request merged**